packages/notebook/src/notebookfooter.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { Widget } from '@lumino/widgets';
+import { Message } from '@lumino/messaging';
+import { Notebook } from './widget';
+import { NotebookActions } from './actions';
+
+/**
+ * A footer widget added after the last cell of the notebook.
+ */
+export class NotebookFooter extends Widget {
+  /**
+   * Construct a footer widget.
+   */
+  constructor(protected notebook: Notebook) {
+    super();
+    this.node.className = 'jp-Notebook-footer';
```

Comment:
Should all caps global const instead of inline CSS class string be used here?

Please see `shell.ts` for reference:
https://github.com/jupyterlab/jupyterlab/blob/82e1a74b61a41fbe5d65e8542053b749cdfe6ea0/packages/application/src/shell.ts#L39